pmda/rmsf.py

```diff
@@ -159,8 +159,10 @@ def _prepare(self):
         self.mean = self.sumsquares.copy()
 
     def _single_frame(self, ts, atomgroups):
-        sumsquares = self.sumsquares
-        mean = self.mean
+        # create new zero-arrays for sumsquares and mean to prevent blocks
```

Comment:
@nawtrey I agree with you that these should be moved to `_reduce` – add a comment saying that all computation is done in `_reduce`. It's a bit of a weird pattern but the value of the RMSF class is really to show us how to adapt the framework to different algorithmic requirements.